### PR TITLE
Update CVE-2015-1828.yml

### PR DESCRIPTION
**Pull request by ixti**

### CODE REVIEW

**Comment by tarcieri** [comment]
:thumbsup:

File: gems/http/CVE-2015-1828.yml [hunk 1]
```diff
@@ -10,3 +10,4 @@ description: |
 cvss_v2: 5
 patched_versions:
   - ">= 0.7.3"
+  - ">= 0.6.4"
```
Comment (tarcieri) [inline gems/http/CVE-2015-1828.yml]: This probably needs to be `"~> 0.6.4"` ?

Comment (ixti) [inline gems/http/CVE-2015-1828.yml]: Oh. Right!

**Comment by ixti** [comment]
Fixed predicate of `0.6.4` version. Thanks @tarcieri.

**Comment by tarcieri** [comment]
As long as you're at it, want to add the OSVDB ID? It's OSVDB 119927

**Comment by ixti** [comment]
@tarcieri I have sent them an e-mail with request to specify version with patch.

**Comment by phillmv** [comment]
OK. Let me know and we'll merge.

**Comment by ixti** [comment]
@phillmv OCVDB have replied:

```                                             
From: j********************g jericho 
Date: Thu, 26 Mar 2015 04:24:27 +0100 
Subject: Re: [OSVDB Mods] Update for OSVDB 119927 

On Thu, 26 Mar 2015, Aleksey Zapparov wrote:

: We have fixed vulnerability in following versions:
: 
: => 0.7.3
: ~> 0.6.4

We had 0.7.3 as a solution, but I have updated to include 0.6.4. Thanks!
```

**Comment by phillmv** [comment]
Cool. Thanks, mate.

**Comment by ixti** [comment]
:bow: :+1:

**Pull request merged**